pycodestyle.py

```diff
@@ -56,6 +56,15 @@
 import time
 import tokenize
 import warnings
+try:
+    from functools import lru_cache
+except ImportError:
+    def lru_cache(maxsize=128):  # noqa as it's a fake implementation.
```

Comment:
According to your [tox.ini](https://github.com/PyCQA/pycodestyle/blob/master/tox.ini), it's 2.6. If the minimum supported version is really 3.3 now, it should probably be dropped from tox.ini?